modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/ClientProtocolCompatibilityTest.cs

```diff
@@ -101,20 +101,34 @@ public void TestClientNewerThanServerReconnectsOnServerVersion()
         {
             // Use a non-existent version that is not supported by the server
             var version = new ClientProtocolVersion(short.MaxValue, short.MaxValue, short.MaxValue);
-            
-            using (var client = GetClient(version))
+
```

Comment:
Please remove this entire test method. It is problematic because of "fake version". I wanted to remove it for a while, now it is time since it causes issues.